### PR TITLE
Adding provider config document

### PR DESCRIPTION
**Pull request by countspongebob**
Document as reviewed/commented in Sig Scalability. @kubernetes/sig-scalability-pr-reviews

### CODE REVIEW

**Comment by countspongebob** [comment]
CLA updated.

File: sig-scalability/provider-configs.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+### Control Plane Cluster Config 
+
+Diagram shows high level target control plan config using the server types listed above, capturing:
+
+* 5 server cluster
+
+* Split etcd across 5 nodes
+
+* API server load balanced
```
Comment (jbeda) [inline sig-scalability/provider-configs.md]: We might want to have hints about this in here.  Like, for example, ELB, by default, has a short timeout that'll cause control plane components to resync often.  Users should set that to the max.

File: sig-scalability/goals.md [hunk 2]
```diff
@@ -70,6 +70,10 @@ NOTES:
   * Goal: 90 minutes
   * Time to restore a fully saturated large cluster is important for cluster-wide failure recovery, and/or related emergency capacity provisioning (e.g. building and populating a new cluster to replace capacity in a failed one). This number also needs to correlate with max pods per cluster, and max scheduler throughput (500,000 pods / 100 pods per second ~ 90 minutes).  We believe that this fulfills most real-world recovery requirements.  The required time to recovery is usually driven primarily by trying to reduce the probability of multiple uncorrelated cluster failures (e.g. "one of our 3 clusters has failed. We're just fine unless another one fails before we've repaired/replaced the first failed one").
 
+## Control Plane Configurations for Testing
+
+Configuration of the control plane for cluster testing varies by provider, and there multiple reasonable configurations. Discussion and guideline of control plane configuration options and standards are documented [here](provider-configs.md).
```
Comment (derekwaynecarr) [inline sig-scalability/goals.md]: nit: and there are multiple

File: sig-scalability/provider-configs.md (hunk 1 above)
Comment (countspongebob) [inline sig-scalability/provider-configs.md]: Hints section added.

**Comment by countspongebob** [comment]
Ready for merge. @spiffxp can you assist?

**Pull request merged**